src/brokers/broker-pods/PodsList.container.tsx

```diff
@@ -36,11 +36,24 @@ export const PodsContainer: FC = () => {
   });
 
   const filterBrokerPods = (pods: K8sResourceCommon[], brokerName: string) => {
-    return pods.filter((pod) =>
-      pod.metadata?.ownerReferences?.some(
-        (ref) => ref.name.startsWith(brokerName) && ref.kind === 'StatefulSet',
-      ),
-    );
+    const BrokerName = brokerName.toLowerCase();
+    return pods.filter((pod) => {
+      if (!pod.metadata?.ownerReferences) {
+        return false;
+      }
+
+      return pod.metadata.ownerReferences.some((ref) => {
+        const refName = ref.name.toLowerCase();
+        const refNameWithoutSuffix = refName.endsWith('-ss')
+          ? refName.slice(0, -3)
+          : refName;
+        if (ref.kind !== 'StatefulSet') {
```

Comment:
This check can be put in front of the refNameWithSuffix extraction in case it is true the extraction operation will be skipped.